<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>669</width>
    <height>396</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Scene Switcher</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Scene Switcher</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_20" stretch="4,5">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QRadioButton" name="websocketRadio">
            <property name="text">
             <string>OBS Studio</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>You must have the obs-websocket plugin installed</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_5" stretch="2,5">
            <item>
             <widget class="QLabel" name="label_2">
              <property name="text">
               <string>IP Address</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="websocketIP"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="2,5">
            <item>
             <widget class="QLabel" name="label_4">
              <property name="text">
               <string>Port</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="websocketPort"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_7" stretch="2,5">
            <item>
             <widget class="QLabel" name="label_6">
              <property name="text">
               <string>Secret</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="websocketSecret"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QRadioButton" name="slobsRadio">
            <property name="text">
             <string>Streamlabs OBS</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>In Streamlabs OBS you must go to Settings, Remote Control and click on the QR Code to enable remote control</string>
            </property>
            <property name="wordWrap">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_19" stretch="2,5">
            <item>
             <widget class="QLabel" name="label_9">
              <property name="text">
               <string>IP Address</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="slobsIP"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_18" stretch="2,5">
            <item>
             <widget class="QLabel" name="label_8">
              <property name="text">
               <string>Port</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="slobsPort"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_21" stretch="2,5">
            <item>
             <widget class="QLabel" name="label_22">
              <property name="text">
               <string>API Token</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLineEdit" name="slobsToken">
              <property name="echoMode">
               <enum>QLineEdit::Password</enum>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_2">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout" stretch="2,5">
            <item>
             <widget class="QLabel" name="label">
              <property name="text">
               <string>In Game</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="inGameCombo"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="2,5">
            <item>
             <widget class="QLabel" name="label_3">
              <property name="text">
               <string>Out of Game</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="outOfGameCombo"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="2,5">
            <item>
             <widget class="QLabel" name="label_5">
              <property name="text">
               <string>Replay</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="replayCombo"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="2,5">
            <item>
             <widget class="QLabel" name="label_7">
              <property name="text">
               <string>Observing</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="observingCombo"/>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QCheckBox" name="switchOnLoading">
            <property name="text">
             <string>Switch to In Game scene on loading screen before game</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="connectButton">
            <property name="text">
             <string>Refresh Scene List</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Extra Scenes</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_17">
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_14" stretch="2,5">
            <item>
             <widget class="QLabel" name="label_14">
              <property name="text">
               <string>Lobby</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="lobbyCombo"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_15" stretch="2,5">
            <item>
             <widget class="QLabel" name="label_12">
              <property name="text">
               <string>User Profile</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="profileCombo"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_9" stretch="2,5">
            <item>
             <widget class="QLabel" name="label_17">
              <property name="text">
               <string>Campaign</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="campaignCombo"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_11" stretch="2,5">
            <item>
             <widget class="QLabel" name="label_21">
              <property name="text">
               <string>Custom Games</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="customCombo"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_12" stretch="2,5">
            <item>
             <widget class="QLabel" name="label_23">
              <property name="text">
               <string>Replays Menu</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="replaysMenuCombo"/>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_3">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_5">
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_16" stretch="2,5">
            <item>
             <widget class="QLabel" name="label_10">
              <property name="text">
               <string>Score Screen</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="scoreScreenCombo"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_8" stretch="2,5">
            <item>
             <widget class="QLabel" name="label_15">
              <property name="text">
               <string>Home</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="homeCombo"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_10" stretch="2,5">
            <item>
             <widget class="QLabel" name="label_19">
              <property name="text">
               <string>Multiplayer</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="multiplayerCombo"/>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_13" stretch="2,5">
            <item>
             <widget class="QLabel" name="label_25">
              <property name="text">
               <string>Collection</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="collectionCombo"/>
            </item>
           </layout>
          </item>
          <item>
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
